Readme.md

```diff
@@ -36,13 +36,15 @@ data = [[ 0.80,  0.55,  0.22,  0.03],
 
 ```python
 from minisom import MiniSom    
-som = MiniSom(6, 6, 4, sigma=0.3, learning_rate=0.5) # initialization of 6x6 SOM
+# initialization of 6x6 SOM with an input of dimension of 4 for training.
+# The initial sigma and learning rate are 0.3 and 0.5, respectively. They decline by dividing (1+t/T) where T is half num_iteration.
+som = MiniSom(6, 6, 4, sigma=0.3, learning_rate=0.5)
 som.train_random(data, 100) # trains the SOM with 100 iterations
 ```
 
-MiniSom implements two types of training. The random training (implemented by the method `train_random`), where the model is trained picking random samples from your data, and the batch training (implemented by the method `train_batch`), where the samples are picked in the order they are stored.
+MiniSom implements two types of training. The random training (implemented by the method `train_random`), where the model is trained picking random samples (i.e. rows) from your data, and the batch training (implemented by the method `train_batch`), where the samples are picked in the order they are stored. Both `train_random` and `train_batch` pick the iteration number of samples (i.e. rows) from your data for training. Ideally, they will approach similar results when iteration number is VERY large. Practically, I recommend using `train_batch` with the iteration number as multiple times of `len(data)` so as to ensure all samples within your data contribute equally.
```

Comment:
I'd rather keep this short as all the details are specified in the documentation of the methods.

Also, I don't see the point in specifying that `train_batch` and `train_random` will yield similar results with enough iterations.

Reply:
> I'd rather keep this short as all the details are specified in the documentation of the methods.

It is surely fine to do this. I just wonder if those new to Python might prefer README.md first.



> Also, I don't see the point in specifying that `train_batch` and `train_random` will yield similar results with enough iterations.

Agreed!